docs/02-project-stellar/05-user-guide/04-custom-dashboard/01-lithiumx/index.md

```diff
@@ -8,7 +8,7 @@ LithiumX is an open source dashboard focused on being simple with a clean UI.
 ## Installation
 Navigate to [https://github.com/Ryzee119/LithiumX/releases](https://github.com/Ryzee119/LithiumX/releases) and download ``LithiumX.zip``.
 
-Extract the contents and transfer the contents to ``E:\Dashboard``.
+Extract the contents to a folder on your hdd. Inside the /bin folder, you will see ``default.xbe``. Transfer default.xbe to ``E:\Dashboard``.
```

Comment:
Can you change ``hdd`` to ``hard drive`` or ``to your system`` to make this a bit more clear?